local/php/php_server.go

```diff
@@ -98,11 +100,18 @@ func (p *Server) Start(ctx context.Context, pidFile *pid.PidFile) (*pid.PidFile,
 	var binName, workerName string
 	var args []string
 	if p.Version.IsFPMServer() {
+		socketDir := path.Join(util.GetHomeDir(), name(p.projectDir))
+
+		if _, err := os.Stat(socketDir); os.IsNotExist(err) {
+			os.MkdirAll(socketDir, os.ModePerm)
+		}
+
+		p.addr = path.Join(util.GetHomeDir(), name(p.projectDir), "php-fpm.sock")
```

Comment:
I would like to see introduced a `fpmSocketFile` method (similar to `fpmConfigFile`)
And also with the same naming strategy that includes the FPM version